DSharpPlus/Net/Rest/DiscordApiClient.cs

```diff
@@ -1785,6 +1785,26 @@ internal async Task<DiscordStageInstance> ModifyStageInstanceAsync(ulong channel
             return stage;
         }
 
+        internal async Task StageInstanceBecomeSpeakerAsync(ulong guildId, ulong id, DateTime? timestamp = null, bool suppress = false, ulong user = 0, string reason = "")
+        {
+            var headers = Utilities.GetBaseHeaders();
+            if (!string.IsNullOrWhiteSpace(reason))
+                headers[REASON_HEADER_NAME] = reason;
+
+
+            var pld = new RestBecomeStageSpeakerInstancePayload
+            {
+                Suppress = suppress,
+                ChannelId = id,
+                RequestToSpeakTimestamp = timestamp
+            };
+            var route = $"guilds/{guildId}{Endpoints.STAGE_INSTANCES}/{(user == 0 ? "@me" : user)}";
```

Comment:
instead of a wonky ternary, consider making `user` nullable and using null-coalescing (`??`) instead? `var user = user?.ToString() ?? "@me";` *should* work.